packages/toggle/UPGRADE.md

```diff
@@ -0,0 +1,7 @@
+# v6 to v7
```

Comment:
AFAICT there's no reason that this shouldn't just be a part of the changelog

Reply:
The problem with Changelogs is once we have 3-4 other minor and patch versions, this information will get lost easily. This is experimental for sure, but I wanted a dedicated place for upgrade guides between major versions to see if that helps folks make those upgrades. I've been listening to feedback that would point to this being useful.

If it turns out to not be that useful, we don't have to do this again.

Reply:
I don't think it makes sense to be adding yet another place where we document changes, it just becomes another burden on us to maintain. While this specific information is relevant it'll likely remain at the top of the changelog and if needed we can direct people to a certain line number explicitly. We'd also then have to document that we even have such files like an UPGRADE.md so that other devs know to check the resource, and since we don't have this for any other component yet I think that may be confusing.

Reply:
Also, as mdb.design will increasingly become the main hub for developer resources, we'd need to think about how this would translate there rather than this information being already available in the changelog.

Reply:
I'm not committing to keeping this, just seeing if this is helpful for the folks who have said it's hard to understand what needs to change for major versions. We don't have to document this anywhere except when communicating this specific major version change.

Also worth noting, many apps don't update to our major versions for 3-6 months in some cases. We can't rely on this information remaining accessible for that long in our changelog.

Reply:
We **don't** need this to be on mongodb.design. For the time being, think of this as a one-off unless this proves significantly beneficial to folks.

Reply:
Ok fine to trial run this, I just worry about maintainability and ensuring that everyone knows about all of the resouces that we have available as it relates to documentation

Reply:
I'm looking for ways to improve our communication, and if this is ineffective or confusing, we won't do it again 😄

Reply:
Sounds good :)